docs/reference/reference-training/experiment-config-reference.rst

```diff
@@ -1300,6 +1300,16 @@ The ``slurm`` section specifies configuration options applicable when the cluste
             - --mem-per-cpu=10
             - --exclusive
 
+``slots_per_node``
+   The minimum number of slots required for a node to be scheduled during a trial. If
```

Comment:
Is it really the "minimum number"?  It's not like we're going to allocate at least that many per node.  That's the exact number we're going to allocate.  Isn't this really:

The number of slots to be allocated per node during a trial.